lib/reporters/mini.js

```diff
@@ -70,19 +70,16 @@ MiniReporter.prototype.prefix = function (str) {
 };
 
 MiniReporter.prototype._test = function (test) {
-	var title;
+	var title = test.title;
 
 	if (test.todo) {
-		title = colors.todo('- ' + test.title);
```

Comment:
But wait, if the next test is slow, user theoretically can actually see skip/todo tests. So I guess we should keep the prefix and color. 

Also, what about changing prefixes to these?

- if `todo` test, prefix = `To-do: `
- if `skip` test, prefix = `Skipped: `